postgres-matrix.yml

```
@@ -0,0 +1,14 @@
+name: Postgres Version Matrix
+project_name: citus # alternatives: citus, citus-enterprise, pg-auto-failover, pg-auto-failover-enterprise
+# There will be one configuration like this for each project in packaging repo
+# i.e. in all-citus, all-enterprise,all-pgautofailover,all-pgautofailover-enterprise, pgxn-citus etc.
```

We usually have comments at the end of a line, or right before it starts.

I think that the comments here are not really needed as this branch is concerned only with citus, and not other possible values of `project_name`
```suggestion
project_name: citus
```

---

Fixed